R/impact_method2.R

```diff
@@ -118,27 +119,45 @@ make_impact <- function(con, index, year_min, year_max, routine_tot_rate_shape =
       shape <- paste(sprintf("AND (year-age) >= %s", cohort_min))
     }
   }
-  focal_countries  <- DBI::dbGetQuery(con, paste("SELECT DISTINCT country",
-                                      "FROM burden_estimate",
-                                      sprintf("WHERE burden_estimate_set = %s",focal$burden_estimate_set_id)))
-
+  
+  if(disease == "HepB") {
```

Comment:
Given the `unique` above, this will not work in general surely?  If `disease` is always a scalar, can you add an assertion right after the `unique(disease)` (just `stopifnot(length(disease) == 1)` is fine).  A comment documenting this would be ideal too

Reply:
Done